tests/examples/example20-kicad-import.test.tsx

```diff
@@ -46,3 +46,52 @@ test(
   },
   20 * 1000,
 )
+
+test(
+  "example20-kicad-direct-import-via-webworker",
+  async () => {
+    const circuitWebWorker = await createCircuitWebWorker({
+      webWorkerUrl: new URL("../../webworker/entrypoint.ts", import.meta.url),
+      platform: {
+        ...getPlatformConfig(),
+        footprintFileParserMap: {
+          kicad_mod: {
+            loadFromUrl: async (url: string) => {
+              console.log("loadFromUrl", url)
+              return {
+                footprintCircuitJson: [],
+              }
+            },
+          },
+        },
+      },
+    })
+
+    try {
+      await circuitWebWorker.executeWithFsMap({
+        fsMap: {
+          "index.tsx": `
+			import kicadMod from "./poly.kicad_mod"
+
+			export default () => {
+			  return (
+				<board width="10mm" height="10mm">
+				  <chip
+					name="U1"
+					footprint={kicadMod}
+				  />
+				</board>
+			  )
+			}
+			`,
+          "poly.kicad_mod": "__STATIC_ASSET__",
+        },
+      })
+
+      await circuitWebWorker.renderUntilSettled()
+    } finally {
+      await circuitWebWorker.kill()
+    }
+  },
+  20 * 1000,
+)
```

Comment:
with eval especially, you really shouldn't modify existing tests to have multiple test files because it greatly increases the number of segfaults